lib/response.ts

```diff
@@ -15,6 +15,12 @@ class WrapError extends Error {
   inner?: any;
 }
 
+const parser = new xml2js.Parser({
```

Comment:
is a global parser reusable?

Reply:
Yes should work.From the readme ...
> You can create one xml2js.Parser per file. That's the recommended one and is promised to always just work.

We do use the parser in one other location in `metadata.ts`. There we don't explicitly create the parser but the lib will internally create an instance.